packages/auth/amplify_auth_cognito/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: amplify_auth_cognito
 description: The Amplify Flutter Auth category plugin using the AWS Cognito provider.
-version: 1.0.0-next.0+4
+version: 1.0.0-next.0+5
```

Comment:
In the changelogs we mentioned `feat(*)`, should be version be `1.0.0-next.1`?

Reply:
It could be either. I was saving `.` bumps for category additions, so `1.0.0-next.1` would be for API/storage/analytics.